azclishell/az_completer.py

```diff
@@ -131,6 +131,7 @@ def get_completions(self, document, complete_event):
 
         text = reformat_cmd(text)
         if text.split():
+
```

Comment:
Unless GitHub is being super weird... how does adding a blank line fix all these issues? 😅